LondonBritishLibrary/orient/BLorient765.xml

```diff
@@ -104,13 +73,137 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                                             <citedRange unit="page">172b-172b</citedRange>
                                         </bibl>
                                     </listBibl>
-                                    
+
                                 </source>
                             </recordHist>
                         </adminInfo>
                     </additional>
+
+<msPart xml:id="p1">
+<msIdentifier></msIdentifier>
+<msContents>
+
+<msItem xml:id="p1_i1">
+<locus target="#3"/>
+<title xml:lang="en" type="incomplete">Part of a prayer</title>
+</msItem>
+
+</msContents>
+
+<physDesc>
+    <objectDesc form="Leaf">
+        <supportDesc>
+            <support>
+                <material key="parchment"/>
+            </support>
+            <extent>
+                <measure unit="leaf">1</measure>
+            </extent>
+        </supportDesc>
+
+        <layoutDesc>
+            <layout columns="2" writtenLines="25" cert="high"/>
+        </layoutDesc>
+    </objectDesc>
+
+    <handDesc>
+
+        <handNote script="Ethiopic" xml:id="h1">
+            <desc>Good hand of the <date>18th century</date>
+            </desc>
+          <date notBefore="1700" notAfter="1800" resp="PRS10171Wright"/>
+            </handNote>
+
+    </handDesc>
+    <additions>
+
+<list>
+
+<item xml:id="e1">
+<desc type="MixedNote">The name of the first owner has been erased, and that of <foreign xml:lang="gez">ክንፈ፡ ሚካኤል፡</foreign>substituted.</desc>
```

Comment:
please encode the name with <persName> and role

Reply:
Done.